doc/enums.tsx

```diff
@@ -34,7 +34,9 @@ export function DocBlockEnum(
         {init && (
           <>
             {" = "}
-            <TypeDef markdownContext={markdownContext}>{init}</TypeDef>
+            <TypeDef typeParams={[]} markdownContext={markdownContext}>
```

Comment:
I assume you are avoiding defaulting `typeParams` to `[]` to ensure you catch all the instances of it being required?  Once you caught them all, it does feel like it could just be an optional param/prop.

Reply:
this is the only place where an empty value is being passed, so i think keeping it like this would be fine